lib/SPIRV/SPIRVLowerMemmove.cpp

```diff
@@ -65,11 +64,6 @@ class SPIRVLowerMemmoveBase : public InstVisitor<SPIRVLowerMemmoveBase> {
       report_fatal_error("llvm.memmove of PHI instruction result not supported",
```

Comment:
I guess, now we can handle this case with `expandMemMoveAsLoop`, right?

Reply:
I guess so, since I haven't seen any disclaimers that `expandMemMoveAsLoop` doesn't work with PHI instructions. I tried to keep the current PR focused on memmove of non-constant length, so I suggest to handle PHI instuctions in a separate patch.

Reply:
@AlexeySotkin , are you ok if I add support for PHI instructions in the next patch?

Reply:
Sure, no problem.